saleor/graphql/product/dataloaders/products.py

```diff
@@ -166,6 +166,35 @@ def batch_load(self, keys):
         return [variant_map.get(product_id, []) for product_id in keys]
 
 
+class AvailableProductVariantsByProductVariantId(
```

Comment:
IMO, for consistency it should be:
```suggestion
class AvailableProductVariantsByProductVariantIdAndChannel
```